src/pages/Guilds/LandingPage.tsx

```diff
@@ -100,8 +102,17 @@ interface TitleProps {
   guildAddress: string;
 }
 const Title: React.FC<TitleProps> = ({ guildAddress }) => {
-  const guildName = useENSNameFromAddress(guildAddress)?.split('.')[0];
-  return <DaoTitle size={2}>{guildName ?? `???`}</DaoTitle>;
+  const ensName = useENSNameFromAddress(guildAddress)?.split('.')[0];
+  const { data } = useGuildConfig(guildAddress);
+  return <DaoTitle size={2}>{ensName ?? data?.name}</DaoTitle>;
+};
+const Proposals: React.FC<TitleProps> = ({ guildAddress }) => {
+  const { data } = useActiveProposalsNow(guildAddress);
+  return (
+    <ProposalsInformation proposals={'active'}>
+      {data?.toString() ?? ''} Proposal

```

Comment:
Is this supposed to show the description of the proposals?. it's not clear what's supposed to be here.

Reply:
`data` ===  number of active proposals. I think is fine. We could say "proposals" if data > 1.

Reply:
Renamed data and also checking for a single proposal